scripts/cli/start_cli_testnet.sh

```diff
@@ -18,11 +18,11 @@ case $1 in
         print_help;exit 0;;
     -r | --release)
         echo "Building and running client in release mode."
-        cargo run -p client --release -- $RUN_PARAMS
+        cargo run -p client --release -- $RUN_PARAMS $FLAGS
```

Comment:
How about we just put "$@" at the end instead of using an env var, that way people can invoke it like `./start_cli_testnet.sh -v` and have it turn on verbose mode?

Reply:
That was my first instinct, but it doesn't jive with the zero-arg invocation captured by the case below.